### PR TITLE
JE-200: Added debtor entity, and debtor options to form.

### PR DESCRIPTION
**Pull request by martinyde**

### CODE REVIEW

File: bundles/GraphicServiceOrder/Repository/DebtorRepository.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * @method Debtor[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class DebtorRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Debtor::class);
+    }
+
+    // /**
```
Comment (tuj) [inline bundles/GraphicServiceOrder/Repository/DebtorRepository.php]: Remove commented out code.

File: bundles/GraphicServiceOrder/Service/OrderService.php [hunk 2]
```diff
@@ -114,26 +115,71 @@ public function __construct(
...
+                $gsOrder
+                    ->setFullName($user->getFullName())
+                    ->setAddress($user->getAddress())
+                    ->setDepartment($user->getDepartment())
+                    ->setPostalcode($user->getPostalCode())
+                    ->setCity($user->getCity());
+            }
         }
-
+        //$response = $this->hammerService->get('/rest/api/2/issue/GSORDER-119');
```
Comment (tuj) [inline bundles/GraphicServiceOrder/Service/OrderService.php]: Remove

File: config/jira_economics.local.yaml.dist [hunk 3]
```diff
@@ -33,7 +33,8 @@ parameters:
         footer: '<p style="margin: 0;">Venlig hilsen<br/> {{ sender.name }}</p>'
 
     gsorder_bundle:
-        field_debitor: customfield_10309
+        field_debitor: 'customfield_10309'
```
Comment (tuj) [inline config/jira_economics.local.yaml.dist:36]: Use 
https://github.com/aakb/jira_economics/blob/develop/src/Service/AbstractJiraService.php#L353
to get custom field ids.

Define them in jira_economics.local.yaml
https://github.com/aakb/jira_economics/blob/develop/config/jira_economics.local.yaml.dist#L3

**Review by tuj — CHANGES_REQUESTED** [review]

File: bundles/GraphicServiceOrder/Repository/DebtorRepository.php (hunk 1 above)
Comment (martinyde) [reply bundles/GraphicServiceOrder/Repository/DebtorRepository.php]: Fixed

File: bundles/GraphicServiceOrder/Service/OrderService.php (hunk 2 above)
Comment (martinyde) [reply bundles/GraphicServiceOrder/Service/OrderService.php]: Fixed

File: config/jira_economics.local.yaml.dist (hunk 3 above)
Comment (martinyde) [reply config/jira_economics.local.yaml.dist:36]: Fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**